android/src/main/java/com/trikot/sample/MainActivity.kt

```diff
@@ -1,23 +1,21 @@
 package com.trikot.sample
 
-import android.os.Bundle
-import androidx.appcompat.app.AppCompatActivity
-import androidx.databinding.DataBindingUtil
-import androidx.lifecycle.ViewModelProviders
-import com.mirego.trikot.viewmodels.LifecycleOwnerWrapper
-import com.trikot.sample.viewmodels.AndroidAppViewModel
+import com.trikot.sample.common.BaseViewModelActivity
 import com.trikot.sample.databinding.ActivityMainBinding
+import com.trikot.sample.viewmodels.base.BaseNavigationDelegate
+import com.trikot.sample.viewmodels.home.HomeViewModel
+import com.trikot.sample.viewmodels.home.HomeViewModelController
 
-class MainActivity : AppCompatActivity() {
-    lateinit var binding: ActivityMainBinding
-    private lateinit var appViewModel: AndroidAppViewModel
+class MainActivity : BaseViewModelActivity<BaseNavigationDelegate, HomeViewModel>() {
```

Comment:
`MainActivity` vs `HomeViewModel`, I feel like those names should be standardized to either `Home` or `Main`